torch/_tensor.py

```diff
@@ -580,6 +580,11 @@ def __len__(self):
             return handle_torch_function(Tensor.__len__, (self,), self)
         if self.dim() == 0:
             raise TypeError("len() of a 0-d tensor")
+        if torch._C._get_tracing_state():
+            warnings.warn('Using len to get tensor shape might cause the trace to be incorrect. '
+                          'Recommended usage would be tensor.shape[0]. '
+                          'Passing a tensor of different shape might lead to errors or silently give '
+                          'incorrect results).', category=torch.jit.TracerWarning, stacklevel=2)
```

Comment:
'Using len to get tensor shape might cause the trace to be incorrect. '
                          'Recommended usage would be tensor.shape[0]. '
                          'Passing a tensor of different shape might lead to errors or silently give '
                          'incorrect results).'

Seems like we missed a '(', or we can remove the last ')'?